cluster/ubuntu/build.sh

```diff
@@ -52,9 +53,9 @@ cp $ETCD/etcd $ETCD/etcdctl binaries/minion
 
 # k8s
 echo "Download kubernetes release ..."
-K8S_VERSION="v0.18.0"
+K8S_VERSION=${K8S_VERSION:-"0.18.0"}
```

Comment:
FYI, you can do this with `: ${VARIABLE:=DEFAULT_VALUE}` too.  The syntax is a little weird, but it doesn't evaluate to X=$X